src/main/resources/swagger.api/schemas/confirmItemArrivalRequest.yaml

```diff
@@ -0,0 +1,9 @@
+ConfirmItemArrivalRequest:
+  description: Confirm item arrival request
+  type: object
+  properties:
+    itemBarcode:
+      description: Barcode number of the item
```

Comment:
description: Barcode of the confirmed item

Reply:
updated